ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/evpn_vlan_bundles.schema.yml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  evpn_vlan_bundles:
+    documentation_options:
+      table: evpn_vlan_bundles
+    type: list
+    primary_key: name
+    convert_types:
+      - dict
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          description: |
+            Specify a EVPN vlan-aware-bundle name.
+            A EVPN vlan-aware-bundle provide a construct to group L2 VLANs and define common settings.
+        id:
+          type: int
+          required: true
+          convert_types:
+            - str
+          min: 1
+          max: 1024
```

Comment:
Remind me why we limit the id?

Reply:
I think there is no real need to limit it, let me remove the limits